docs/faq.md

```diff
@@ -122,5 +122,67 @@ user is in control. Windows and views created by the app are positioned by the
 OS shell. Its UX is either drawn using XAML or Composition objects or a
 framework that produces XAML or Composition objects.
 
-## I don't see my question here! [Create an issue to ask a question or start a discussion](https://github.com/microsoft/ProjectReunion/issues/new/choose).
+## How will I consume Project Reunion components? How will they be packaged?
+
```

Comment:
What're these edits from?

Reply:
I am not sure, but will be moving to a different PR ultimately, so this shouldn't be an issue